src/behaviors/scrollable.js

```diff
@@ -13,11 +13,15 @@ export default function scrollable(WrappedComponent) {
     }
 
     componentDidMount() {
-      this.node.addEventListener('scroll', this.updateScrollState);
+      if (this.node) {
```

Comment:
Originally this component used ::before ::after classes that were animated in depending on the scrolling state from the dom. Now the fade is a constant, so we don't need to keep track of the scrolling state.